src/libraries/illink-oob.targets

```diff
@@ -4,7 +4,8 @@
           DependsOnTargets="PrepareForAssembliesTrim">
 
     <PropertyGroup>
-      <OOBAssembliesTrimmedArtifactsPath>$([MSBuild]::NormalizePath('$(ILLinkTrimAssemblyArtifactsRootDir)', 'trimmed-oobs'))</OOBAssembliesTrimmedArtifactsPath>
+      <OOBAssembliesTrimmedArtifactsPath>$([MSBuild]::NormalizeDirectory('$(ILLinkTrimAssemblyArtifactsRootDir)', 'trimmed-oobs'))</OOBAssembliesTrimmedArtifactsPath>
+      <OOBAssembliesMarkerFile>$(BaseIntermediateOutputPath)oob-linker-marker.txt</OOBAssembliesMarkerFile>
```

Comment:
Does this path encode sufficient information such that it is unique for a given set of inputs? EG: configuration, target framework, runtime etc -- not sure all the variables going into the inputs.  Perhaps sharing sample values of the inputs and outputs would help confirm.

Reply:
It doesn't encode any of the properties you mentioned above. Would it be better to use IntermediateOutputPath instead?
The inputs are dependent on these, you are right: `NetCoreAppCurrentRuntimePath = C:\git\runtime3\artifacts\bin\runtime\net7.0-windows-Debug-x64\`